DoorGate/DoorInteractor.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  DoorInteractor.swift
+//  DoorGate
+//
+//  Created by Yuliani Noriega on 1/29/18.
+//  Copyright © 2018 Yuliani Noriega. All rights reserved.
+//
+
+import UIKit
+
+class DoorInteractor: NSObject {
+
```

Comment:
empty interactor. Was this not necessary for the exercise? If so, lets kill, if not, let's get it added?

Reply:
yes you are right, i had deleted it from xcode but not from the project directory, good catch